Classes/Controller/CalendarController.php

```diff
@@ -12,33 +12,42 @@
 namespace JWeiland\Events2\Controller;
 
 use JWeiland\Events2\Helper\CalendarHelper;
+use JWeiland\Events2\Traits\TypoScriptFrontendControllerTrait;
+use Psr\Http\Message\ResponseInterface;
 use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
 
 /*
  * Controller to show the LiteCalendar. Further, it stores the selected month in user-session
  */
 class CalendarController extends AbstractController
 {
+    use TypoScriptFrontendControllerTrait;
+
     protected CalendarHelper $calendarHelper;
 
     public function injectCalendarHelper(CalendarHelper $calendarHelper): void
     {
         $this->calendarHelper = $calendarHelper;
     }
 
-    public function showAction(): void
+    public function showAction(): ResponseInterface
     {
         $frameworkConfiguration = $this->configurationManager->getConfiguration(
```

Comment:
is it better to move this to a seperate function?

Reply:
I have moved it into an own method now, because of better possibility of explaination.